src/EditorFeatures/VisualBasicTest/ConvertForEachToFor/ConvertForEachToForTests.vb

```diff
@@ -395,7 +395,67 @@ Class Test
     End Sub
 End Class
 "
-            Await TestMissingInRegularAndScriptAsync(initial)
+            Dim expected = "
+Class Test
+    Sub Method()
+        Dim {|Rename:array|} = New Integer() {1, 2, 3}
+        For {|Rename:i|} = 0 To array.Length - 1
```

Comment:
@Paxxi What does multiple rename annotations mean in the expected baseline?

Reply:
The source enumerable is defined inline so it doesn't have a name, the conversion prompts to give the source a name and rename the loop variable.
I've taken these from the other tests in the same file.